ni/github.py

```diff
@@ -89,6 +91,12 @@ def __init__(self, server, event, request):
             raise abc.ResponseExit(
                     status=http.HTTPStatus.UNSUPPORTED_MEDIA_TYPE, text=msg)
 
+        binary_content = await request.read()
+
+        if not Host._verify_signature(server.contrib_secret_token(), binary_content,
+                                      request.headers['HTTP_X_HUB_SIGNATURE']):
+            raise abc.ResponseExit(status=http.HTTPStatus.UNAUTHORIZED)
```

Comment:
We should log a warning that the signature check failed and then comment out the exception raising. That way it's easy to verify things are working before we flip this on permanently.